src/DynamoCoreWpf/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="DynamoVisualProgramming.LibG_228_0_0" version="2.12.0.6565" targetFramework="net48" developmentDependency="true" />
+  <package id="Greg" version="2.1.7971.20260" targetFramework="net48" />
+  <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net48" />
+  <package id="RestSharp" version="106.12.0" targetFramework="net48" />
```

Comment:
Shouldn't this be `106.12.0.0` as above?

Reply:
will check, nuget may equate those two version numbers.